Model/Behavior/UploadBehavior.php

```diff
@@ -1028,10 +1022,10 @@ protected function _getThumbnailName($configuredName, $usePrefixStyle) {
 		}
 
 		if ($usePrefixStyle) {
-			return '{size}_{filename}';
+			return '{size}{filename}';
```

Comment:
Im sorry Jose i use it to dont let a original copy on folder ... it was fixed now thx